content/200-orm/200-prisma-client/450-testing/100-unit-testing.mdx

```diff
@@ -135,11 +135,11 @@ to your `tsconfig.json`.
    })
    ```
 
-This will create a new context before each test is run via the `createMockContext` function. This (`mockCtx`) context will be used to make a mock call to Prisma and run a query to test. The `ctx` context will be used to run a scenario query that is tested against.
+This will create a new context before each test is run via the `createMockContext` function. This (`mockCtx`) context will be used to make a mock call to Prisma ORM and run a query to test. The `ctx` context will be used to run a scenario query that is tested against.
```

Comment:
```suggestion
This will create a new context before each test is run via the `createMockContext` function. This (`mockCtx`) context will be used to make a mock call to Prisma Client and run a query to test. The `ctx` context will be used to run a scenario query that is tested against.
```